backend_entrypoint.sh

```diff
@@ -20,6 +20,7 @@ cmd_init() {
     ~/manage.py migrate
 
     wait-for-it "${CVAT_REDIS_INMEM_HOST}:${CVAT_REDIS_INMEM_PORT:-6379}" -t 0
+    ~/manage.py migrateredis
```

Comment:
The logic in this script also waits for migrations to complete before starting the backend services. Perhaps we should implement something similar for Redis migrations?

Reply:
I'll check

Reply:
done